pkg/v7/configmap/desired_test.go

```diff
@@ -142,14 +142,21 @@ func Test_ConfigMap_GetDesiredState(t *testing.T) {
 				Namespaces: []string{},
 			},
 			configMapValues: ConfigMapValues{
+				CalicoAddress:                     "172.20.0.0",
```

Comment:
For the tests I think the desired test should just checks the correct configmaps are created. So these values don't need to be set.

Each values function has its own test. But maybe the values funcs should also validate the settings? This is done in the provider resources but its easy to get the wiring wrong. WDYT?

Reply:
Yes lets go with the current validation in the resource. I was over thinking this one! 😄

Reply:
About the values for tests we still need to set a valid `ClusterIPRange`, the DNS IP is calculated from it here https://github.com/giantswarm/cluster-operator/blob/master/pkg/v7/configmap/desired.go#L86. Readded.